eras/mary/impl/src/Cardano/Ledger/Mary/Translation.hs

```diff
@@ -117,7 +114,7 @@ instance Crypto c => TranslateEra (MaryEra c) ShelleyTxOut where
   translateEra () (TxOutCompact addr cfval) =
     pure $ TxOutCompact (coerce addr) (translateCompactValue cfval)
 
-instance Crypto c => TranslateEra (MaryEra c) UTxO where
+instance (Crypto c) => TranslateEra (MaryEra c) UTxO where
```

Comment:
I thought you were the destroyer of extraneous parenthesis :laughing:

Reply:
I am! I have no idea how I could commit such an atrocity!
:smile: 

Fixed